File: C++/061. Espelho/Espelho.cpp
#include <iostream>
int main()
{
    int A, B;
    std::cin >> A >> B;

    if (A / 1000 == B % 10 && (A / 100) % 10 == (B / 10) % 10 && (A / 10) % 10 == (B / 100) % 10 && A % 10 == B / 1000)
    {
        std::cout << "espelho" << std::endl;
        return 0;
    }
    else
    {
        std::cout << "nao espelho" << std::endl;
        return 0;
    }
   return 0;
}

File: Java/061. Espelho/Espelho.java
import java.util.Scanner;

public class Espelho 
{
    public static void main(String[] args) 
    {
        Scanner scan = new Scanner(System.in);
        int A = scan.nextInt();
        int B = scan.nextInt();
        String aStr = String.valueOf(A);
        String bStr = String.valueOf(B);
        String bReverso = new StringBuilder(bStr).reverse().toString();

        if (aStr.equals(bReverso)) {
            System.out.println("espelho");
        } else {
            System.out.println("nao espelho");
        }
        scan.close();
    }
}